Simplenote/MagicLinkRequestedView.swift

```diff
@@ -0,0 +1,97 @@
+import Foundation
+import SwiftUI
+
+
+// MARK: - MagicLinkConfirmationView
+//
+struct MagicLinkRequestedView: View {
+
+    @State private var displaysFullImage: Bool = false
+    let email: String
+    var onDismissRequest: (() -> Void)?
+
+    
+    private var emailLengthLimited: String {
+        guard email.count > Metrics.maximumEmailLength else {
+            return email
+        }
+        
+        return String(email.prefix(Metrics.maximumEmailLength)) + "..."
+    }
+
+    var body: some View {
+        VStack(alignment: .center, spacing: 10) {
+            Image(systemName: MagicLinkImages.mail)
+                .renderingMode(.template)
+                .font(.system(size: 48))
+                .foregroundColor(Color(nsColor: .simplenoteBrandColor))
+                .scaleEffect(displaysFullImage ? 1 : 0.4)
+                .onAppear {
+                    withAnimation(.spring(response: 0.3, dampingFraction: 0.3)) {
+                        displaysFullImage = true
+                    }
+                }
+                .padding(.bottom, Metrics.imagePaddingBottom)
+                .padding(.top, Metrics.imagePaddingTop)
+
+            Text("Check your email")
+                .bold()
+                .font(.title)
+                .padding(.bottom, Metrics.titlePaddingBottom)
+                        
+            Text("If an account exists, we've sent an email with a link that'll log you in to **\(emailLengthLimited)**")
+                .font(.title3)
+                .lineLimit(nil)
+                .multilineTextAlignment(.center)
+                .padding()
+            
+            SwiftUI.Button(action: switchToAuthenticationUI) {
```

Comment:
Do you need `SwiftUI.Button`?  Can't it just be `Button`?

Reply:
Unfortunately, there's another class named `Button`, so both collide 💔